### ISSUE TITLE
unicode error with metadata import

### ISSUE THREAD

**Issue by colinallen** [inpho/topic-explorer#172]
```python
% topicexplorer metadata jeff.ini -a labels.csv --rename
Traceback (most recent call last):
  File "/Users/colin/anaconda/bin/topicexplorer", line 9, in <module>
    load_entry_point('topicexplorer', 'console_scripts', 'topicexplorer')()
  File "/Users/colin/topicexplorer/topicexplorer/**main**.py", line 243, in main
    benchmark(metadata.main)(args)
  File "/Users/colin/topicexplorer/topicexplorer/metadata.py", line 138, in main
    metadata = parse_metadata_from_csvfile(args.add, context_type)
  File "/Users/colin/topicexplorer/topicexplorer/metadata.py", line 29, in parse_metadata_from_csvfile
    for row in reader:
  File "/Users/colin/topicexplorer/topicexplorer/metadata.py", line 16, in UnicodeDictReader
    for key, value in row.iteritems()}
  File "/Users/colin/topicexplorer/topicexplorer/metadata.py", line 16, in <dictcomp>
    for key, value in row.iteritems()}
TypeError: coercing to Unicode: need string or buffer, list found
```

**Comment by JaimieMurdock** [comment]
We are having another issue with this in the Handian corpus. Investigating how to make this work.

**Comment by JaimieMurdock** [comment]
Tracking this down by working with issue #104 as well to have better typing to reduce the quality.

**Comment by JaimieMurdock** [comment]
Corrected with following patch in d91dbdb3:
```python
def parse_value(value):
    try:
        return literal_eval(value)
    except SyntaxError:
        return value
```
